.gitmodules

```diff
@@ -1,3 +1,9 @@
-[submodule "dapple_packages/erc20"]
-	path = dapple_packages/erc20
+[submodule "lib/erc20"]
+	path = lib/erc20
 	url = https://github.com/nexusdev/erc20
```

Comment:
`https://github.com/nexusdev/erc20` is also moved to `https://github.com/dapphub/erc20`